Makefile

```diff
@@ -27,7 +27,7 @@ EXTRA_MANIFESTS_PATH = ./edge_configs/extra-manifests
 MACHINE_NETWORK ?= 192.168.127.0/24
 CPU_CORE ?= 16
 RAM_MB ?= 32768
-
+IBU_IMAGER ?= quay.io/lochoa/ibu-imager:latest
```

Comment:
better to use `quay.io/openshift-kni/lifecycle-agent-operator:latest`

(given that active development is taking place there from now on)